torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -542,7 +542,7 @@ class CppMicroGemmAMX(CppMicroGemm):
         {%- set tile_idx_b = tile_offset_b + tile_col %}
         {%- set tile_idx_c = tile_row * num_columns + tile_col %}
         {%- if tile_col == 0 %}
-        _tile_loadd({{tile_idx_a}}, A + {{tile_row * 16}} * lda + k, lda * sizeof({{input_t}}));
+        _tile_stream_loadd({{tile_idx_a}}, A + {{tile_row * 16}} * lda + k, lda * sizeof({{input_t}}));
```

Comment:
Can we separate this change to another PR? Or the perf gain from this depends on cache blocking?

Reply:
Sure, moved this change to another PR: https://github.com/pytorch/pytorch/pull/129455